babel.config.js

```diff
@@ -3,5 +3,6 @@ module.exports = {
   plugins: [
     // react-native-reanimated/plugin has to be last
     'react-native-reanimated/plugin',
+    'transform-inline-environment-variables',
```

Comment:
just to account for the comment in this file, move this so it's before the other plugin